server/config/forms/relocationAndInjuriesForm.js

```diff
@@ -29,6 +29,12 @@ const completeSchema = joi.object({
     otherwise: joi.any().strip(),
   }),
 
+  typeOfRelocation: joi.when('relocationType', {
```

Comment:
bit confusing having `relocationType` and `typeOfRelocation`

Could the name of the new field be something like `otherRelocationDescription` or something?

Reply:
`userSpecifiedRelocationType` ? 
...
something else?

Reply:
Implemented as suggested - we went with userSpecifiedRelocationType.